doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_cartodb_sink.md

```diff
@@ -0,0 +1,455 @@
+* Data model by entity (`data_model=dm-by-entity`). For each entity, the notified/default FIWARE service path is concatenated to the notified entity ID and entityType in order to compose the table name. The concatenation charated is `_` (underscore). If the FIWARE service path is the root one (`/`) then only the entity ID and type are concatenated.
+* Data model by attribute (`data_model=dm-by-attribute`). For each entity and entity's attribute, the notified/default FIWARE service path, the notified entityId and entityType, and the attribute name and type are concatenated in order to compose the table name. The concatenation charated is `_` (underscore). If the FIWARE service path is the root one (`/`) then only the entity ID and type and the attribute name and type are concatenated.
+
+It must be said PostgreSQL only accepts alphanumeric characters and the underscore (`_`). All the other characters will be escaped to underscore (`_`) when composing the table names.
+
+The following table summarizes the table name composition:
+
+| FIWARE service path | `dm-by-service-path` | `dm-by-entity` | `dm-by-attribute` |
+|---|---|---|---|
+| `/` | N/A | `<entityId>_<entityType>` | `<entityId>_<entityType>_<attrName>_<attrType>` |
+| `/<svcPath>` | `<svcPath>` | `<svcPath>_<entityId>_<entityType>` | `<svcPath>_<entityId>_<entityType>_<attrName>_<attrType>` |
+
+[Top](#top)
+
+####<a name="section1.2.3"></a>Raw-based storing
+Regarding the specific data stored within the tables, if `enable_raw` parameter is set to `true` (default storing mode) then the notified data is **stored as it is, without any processing or modification**. This is the simplest way of storing geolocation data.
+
+A single insert is composed for each notified entity, containing such insert the following fields:
+
+* `recvTime`: UTC timestamp in human-redable format ([ISO 8601](http://en.wikipedia.org/wiki/ISO_8601)).
+* `fiwareServicePath`: Notified fiware-servicePath, or the default configured one if not notified.
+* `entityId`: Notified entity identifier.
+* `entityType`: Notified entity type.
+* `the_geom`: The current geolocation point, i.e. containing longitude and latitude. It must be of type PostGIS Geometry, it can be created for instance using [`ST_Point()`](http://postgis.net/docs/ST_Point.html). The data used for composing the PostGIS Geometry is obtained from a special notified attribute:
+    * Either of type `geo:point`.
+    * Either having associated a `location` metadata of type `string` and value `WGS84`.
+* For each not-geolocated attribute, the insert will contain two additional fields, one for the value, named `<attrName>`, and another for the metadata, named `<attrName>_md`).
```

Comment:
Just out of my curiosity, is the same information stored no mater the data model? Or depending on the data model the information included in the table names is not included?